SplunkAppForWazuh/appserver/static/js/controllers/agents/inventory/inventory.html

```diff
@@ -93,7 +93,7 @@
       <span class="wz-headline-title"><i class="fa fa-fw fa-exchange"></i> Network ports</span>
       <span class="color-grey pull-right">Last scan: {{syscollector.ports.items[0].scan.time}}</span>
       <md-divider class="wz-margin-top-10"></md-divider>
-      <wazuh-table flex path="'/syscollector/' + agent.id + '/ports'" row-sizes="[4]" keys="[{value:'local.ip',nosortable:true},{value:'remote.ip',nosortable:true},'state','protocol']">
+      <wazuh-table flex path="'/syscollector/' + agent.id + '/ports'" row-sizes="[4]" keys="[{value:'local.ip',nosortable:true},{value:'local.port',nosortable:true},{value:'remote.ip',nosortable:true},'state','protocol']">
```

Comment:
`remote.port` is missing.

Also check if those fields are sortable please.

Reply:
They're not currently sortable.